lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
+    '''tests for the long_int_(mathfunction) functions'''
+    def setUp(self):
+        self.long_big = 10**1000
+        self.float_big = 1.0e+300
+        self.float_sm = 1.0e-100
+
+    def tearDown(self):
+        del self.long_big
+        del self.float_big
+        del self.float_sm
+
+    def error_tester(self, result, expected):
```

Comment:
Perhaps I'm wrong, but it looks like you should be using assertAlmostEqual(a, b) which allows you to give the number of places and a delta.

If you keep this method the name should begin with assert and say what it's asserting (ie how it is different)